GDJS/Runtime/runtimegame.js

```diff
@@ -548,3 +555,11 @@ gdjs.RuntimeGame.prototype.wasFirstSceneLoaded = function() {
 gdjs.RuntimeGame.prototype.getSceneStack = function() {
   return this._sceneStack;
 }
+
+/**
+ * Checks if the IDE marked the current build as preview.
```

Comment:
nitpicking: `Checks if the game is running as a preview, launched from an editor.`

("build" is not a term that we employ in previews).